### DISCUSSION TITLE
Wat te doen met devices zonder sensoren?

### DISCUSSION THREAD

**Discussion by CelineJansen** [kadaster-labs/sensrnet-home#243] [Pilot Gemeente]
Ben jij een pilotdeelnemer of gebruiker? Dan hebben wij vanuit het FAB een discussiepunt voor jou!

**Wat te doen met devices zonder sensoren?**

Op dit moment kan een device 0, 1, of meerdere sensoren hebben (en sensoren hebben vervolgens weer 0, 1, of meerdere datastromen).

Het kan zorgen voor vervuiling van de data, en verwarring bij gebruikers wanneer er devices zonder sensoren terug te vinden zijn in de viewer. Aan de andere kant kan het praktisch zijn om alvast een device op te kunnen voeren ter voorbereiding, of wanneer een sensor (tijdelijk) buiten gebruik is het device wel te laten staan. 
Wat is jouw mening hierover? Wel of geen devices zonder sensoren zichtbaar in de viewer en/of toestaan in het datamodel?

**Comment by marcvanandel** [comment]
Een device is wat ik als burger onder 'sensor' beschouw. De sensor in het datamodel is voor de technische details zeker relevant ... maar voor mij als burger niet (per se). Allereerst wil ik weten wat daar hangt en dan zijn de categorieën van device informatief genoeg (denk ik). Als er ergens een 'sensor' (device dus) hangt, dan zou deze op de kaart zichtbaar moeten zijn, denk ik.

Ik vind het wel raar om een leeg tabje onder sensor te zien. Zouden we de informatie anders moeten tonen ... en verbergen wat er niet is?
